gpjax/kernels/computations/eigen.py

```diff
@@ -33,14 +36,15 @@ class EigenKernelComputation(AbstractKernelComputation):
     """
 
     def cross_covariance(
-        self, x: Num[Array, "N D"], y: Num[Array, "M D"]
+        self, kernel: Kernel, x: Num[Array, "N D"], y: Num[Array, "M D"]
```

Comment:
Noticed some of the signatures for `cross_covariance` and `gram` use `Num` instead of `Float`. Not sure if these were intended (or how they crept in if not)? But the were here prior to your PR, so can be addressed in a separate issue.

Reply:
Yes I was also confused about that, and apparently it's because the `GraphKernel` implementation accepts integer types. I had all `Num`s changed to `Float` but beartype was complaining so I had to switch back...for now 😅